backend/apps/accounts/migrations/0015_add_jwt_token_version.py

```
@@ -0,0 +1,25 @@
+"""
+Migration to add jwt_token_version to UserProfile.
+"""
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '000X_previous_migration'),  # Replace with actual previous migration
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)